### ISSUE TITLE
Run shed as a python module

### ISSUE THREAD

**Issue by whatnick** [Zac-HD/shed#14]
In use cases where python is part of an application install such as blender it is useful to run shed as a python module. Currently the result is as below.
```
 .\python.exe -m shed
C:\Program Files\Blender Foundation\Blender 2.92\2.92\python\bin\python.exe: No module named shed.__main__; 'shed' is a package and cannot be directly executed
```

**Comment by Zac-HD** [comment]
Fixed in version 0.3.3, https://github.com/Zac-HD/shed/commit/ab1d28c8fd7284e4aeefe30bfecf5e32b54f75c3

**Comment by whatnick** [comment]
Thanks for the super quick fix. I will get you some stickers from my project.
